webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/repository/ldap/LDAPSwapGroupLibrary.scala

```diff
@@ -259,7 +259,7 @@ class ImportGroupLibraryImpl(
         }
       }
 
-      def recSanitizeCategory(content:NodeGroupCategoryContent, parent: NodeGroupCategory, isRoot:Boolean = false) : Option[NodeGroupCategoryContent] = {
+      def recSanitizeCategory(content: NodeGroupCategoryContent, parent: NodeGroupCategory, isRoot: Boolean) : Option[NodeGroupCategoryContent] = {
```

Comment:
can't we have default value anymore?

Reply:
default in recursive methods lead to a warning saying "be careful, it's using a default value!", I'm not sure about the intent but I wasn't in a mood to fight scalac